packages/navigation/src/index.js

```diff
@@ -61,6 +61,16 @@ export const getQueryExcludedScreens = () =>
 		'homescreen',
 	] );
 
+/**
+ * Given a path, return whether it is an excluded screen
+ *
+ * @param {Object} path Path to check
+ *
+ * @return {boolean} Boolean representing whether path is excluded
+ */
+export const pathIsExcluded = ( path ) =>
+	getQueryExcludedScreens().includes( getScreenFromPath( path ) );
```

Comment:
Just thinking out loud here (and definitely not within the scope of this PR)- wondering if this should have been included screens instead of excluded.  May be hard to maintain as more WCA pages are registered.

Reply:
💯 . I was actually thinking this last time I was working on persisted queries, but was a little uncertain since there is a filter associated with it, `TIME_EXCLUDED_SCREENS_FILTER`, that I assume we'd still have to support.

I definitely think a whitelist would be more appropriate here.